src/styles/index.js

```diff
@@ -1,18 +1,45 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import * as PropTypes from 'prop-types';
 import { createGlobalStyle, ThemeProvider } from 'styled-components';
 
 const GlobalStyle = createGlobalStyle`
   html {
     font-size:1vw;
     color: ${props => props.theme.primaryColor};
     background: ${props => props.theme.backgroundColor};
-    overflow: hidden;
+  }
+  html, body {
     height: 100%;
+    margin:0;
+    overflow: hidden;
   }
   a {
     text-decoration: none;
   }
+  /* only apply in webkit */
+  @media screen and (-webkit-min-device-pixel-ratio:0) { 
+    .iceberx-app-container {
+        color: rgba(0,0,0,0);
+        text-shadow: 0 0 black;
+        box-shadow: 0 1px 6px black;
+        transition: color .8s;
+    }
+    .iceberx-app-container:hover {
+       color: rgba(0,0,0,0.3);  
```

Comment:
It seems when hover on the window the hover will take effect in Mac.

Reply:
Yep. This is some tricky part. Scrollbar itself does not have the transition effect. Check [here](https://stackoverflow.com/questions/19230289/use-transition-on-webkit-scrollbar).

Reply:
It's okay for now. If no better solution, the implementation is good enough. Just try to make it Windows specific.